en/docs/develop/product-apis/publisher-apis/publisher-v1/publisher-v1.yaml

```diff
@@ -2520,23 +2520,12 @@ paths:
         200:
           description: |
             OK.
-            Successful response if the api name exist.
-          schema:
-            $ref: '#/definitions/Document'
-          headers:
-            Content-Type:
-              description: |
-                The content type of the body.
-              type: string
-            ETag:
-              description: |
-                Entity Tag of the response resource.
-                Used by caches, or in conditional requests (Will be supported in future).
-              type: string
+            Successful response if the document name exist.
         400:
           $ref: '#/responses/BadRequest'
-        404:
-          $ref: '#/responses/NotFound'
+        404: |
+          Not Found.
```

Comment:
Please make sure to move this changes to REST API definitions in carbon-apimgt

Reply:
added with  https://github.com/wso2/carbon-apimgt/pull/9528